src/components/Checkbox.vue

```diff
@@ -1,4 +1,63 @@
-@import '../../scss/variables/colors';
+<template lang="pug">
+#checkbox
+  .container-options
+    .control-form
+      label(for='primaryRadioColor') Primary Color
+      input#primaryRadioColor.input-control(type='color' value='#636060')
+    .control-form
+      label(for='secondaryRadioColor') Secondary Color
+      input#secondaryRadioColor.input-control(type='color' value='#FFFFFF')
+    .control-form
+      label(for='activeRadioColor') Active Color
+      input#activeRadioColor.input-control(type='color' value='#00D478')
+    .control-form
+      label(for='radioBackgroundColor') Background Color
+      input#radioBackgroundColor.input-control(type='color' value='#333333')
+  .container-checkbox
+    .item-checkbox
+      .checkbox-1
+        input#checkbox-1.checkbox(type='checkbox' name='checkbox-1')
+        label(for='checkbox-1')
+    .item-checkbox
+      .checkbox-2
+        input#checkbox-2.checkbox(type='checkbox')
+        label(for='checkbox-2')
+          svg(width='18px' height='18px' viewbox='0 0 18 18')
+            path(d='M1,9 L1,3.5 C1,2 2,1 3.5,1 L14.5,1 C16,1 17,2 17,3.5 L17,14.5 C17,16 16,17 14.5,17 L3.5,17 C2,17 1,16 1,14.5 L1,9 Z')
+            polyline(points='1 9 7 14 15 4')
+    .item-checkbox
+      .checkbox-3
+        input#checkbox-3.checkbox(type='checkbox')
+        label(data-off='OFF' data-on='ON' for='checkbox-3')
+    .item-checkbox
+      .checkbox-4
+        input#checkbox-4.checkbox(type='checkbox')
+        label(for='checkbox-4')
+    .item-checkbox
+      .checkbox-5
+        input#checkbox-5.checkbox(type='checkbox')
+        label(data-off='Nope' data-on='Yeah!' for='checkbox-5')
+    .item-checkbox
+      .checkbox-6
+        input#checkbox-6.checkbox(type='checkbox')
+        label(for='checkbox-6') Checkbox 6
+    .item-checkbox
+      .checkbox-7
+        input#checkbox-7.checkbox(type='checkbox')
+        label(for='checkbox-7') Checkbox 7
+    .item-checkbox
+      .checkbox-8
+        input#checkbox-8.checkbox(type='checkbox')
+        label(for='checkbox-8')
+</template>
+<script>
+import '../../scripts/checkbox.js'
+
+export default {}
+</script>
+
+<style lang="scss" scoped>
+  @import '../scss/variables/_colors';
 $transition-duration: 330ms;
```

Comment:
I think that this is used in many places. Is it better to move it to `variables.scss` file or not?